src/MainWindow.vala

```diff
@@ -220,12 +225,24 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
         refresh_menuitem.get_child ().destroy ();
         refresh_menuitem.add (refresh_accellabel);
 
+        var repair_accellabel = new Granite.AccelLabel.from_action_name (
+            _("Repair"),
+            "app.repair"
+        );
+
+        repair_menuitem = new Gtk.ModelButton () {
+            action_name = "app.repair"
+        };
+        repair_menuitem.get_child ().destroy ();
+        repair_menuitem.add (repair_accellabel);
+
         var menu_popover_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 0) {
             margin_bottom = 6,
             margin_top = 6
         };
         menu_popover_box.add (automatic_updates_button);
         menu_popover_box.add (refresh_menuitem);
+        menu_popover_box.add (repair_menuitem);
```

Comment:
I'm not sure this makes sense to show preemptively. If we give the option for repairing when there's an error, it kind of seems like this menu item wouldn't really be used or it would be confusing why it's there and how to make it go away